.github/workflows/ci.yml

```
@@ -0,0 +1,52 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.nightly }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
+          - '1.3'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+        nightly: [false]
+        include:
+          - os: ubuntu-latest
+            version: 'nightly'
+            nightly: true
```

This seems a bit redundant, doesn't it? If you want to figure out if something runs with Julia nightly you could just check `matrix.version`:
```suggestion
        include:
          - os: ubuntu-latest
            version: 'nightly'
```

---

Ah you mean I could write `continue-on-error: ${{ matrix.version == 'nightly' }}` ?

---

Yes

---

That's what we do in other Turing packages and it works :smiley:

---

Nice!